.github/workflows/sycl_nightly.yml

```
@@ -34,7 +34,7 @@ jobs:
     secrets: inherit
     with:
       build_cache_root: "/__w/"
-      build_artifact_suffix: default
+      build_artifact_suffix: default-2204
```

I have a slight preference to 
```suggestion
      build_artifact_suffix: ubuntu2204
```

to align with the step's name, but that is subjective.

---

"default" here means build configuration. In addition to "default" we have "no-assert" build and "self-build".